node/src/chain_specs/dolphin.rs

```diff
@@ -28,6 +28,9 @@ const DOLPHIN_PROTOCOL_ID: &str = "dolphin"; // for p2p network configuration
 const KUSAMA_RELAYCHAIN_LOCAL_NET: &str = "kusama-local";
 const KUSAMA_RELAYCHAIN_DEV_NET: &str = "kusama-dev";
 
+/// The default XCM version to set in genesis config.
+const SAFE_XCM_VERSION: u32 = xcm::prelude::XCM_VERSION;
```

Comment:
I think we should make the version number explicit, instead relying on some external defined version number.